### PR TITLE
Add recommendation and rationale for labels on app creation; add link to CLI Reference.

### PR DESCRIPTION
**Pull request by tnguyen-rh**
@kargakis @sosiouxme 
I think "osc new-app" is the right place to mention labels in the context of easy deletion, and it mentions not only deletion, but selection and (general) manipulation as well.  WDYT?

### CODE REVIEW

File: dev_guide/new_app.adoc [hunk 1]
```diff
@@ -68,8 +68,11 @@ path:
 $ osc new-app <myregistry.com>/<mycompany>/<image_name>
 ----
 
-You can also create an application from a remote repository using a label. For 
-example, to create an application from the `ruby-hello-world` remote repository:
+You can also create an application from a remote repository using a label.
```
Comment (sosiouxme) [inline dev_guide/new_app.adoc]: This kind of implies the label has some part in creating the app, or that it's somehow specific to creating from a remote repo. I'd say something like "You can also apply a label when creating an application."

Comment (kargakis) [inline dev_guide/new_app.adoc]: Agreed. This is spelled out wrong and needs to change.

**Comment by tnguyen-rh** [comment]
@sosiouxme @kargakis 
Thanks for the feedback.  I have reworded that sentence and made a parallel change to the example-intro sentence, as a result.

**Comment by kargakis** [comment]
LGTM

**Comment by tnguyen-rh** [comment]
@bfallonf @tpoitras @adellape 
If you get a chance, could you please review this PR?

**Comment by adellape** [comment]
LGTM

**Pull request merged**